### PR TITLE
Add Project Readme, OCA style

### LINKED ISSUE #38: Improve Readme
Hi! In @coopdevs we will test vertical-cooperative with Odoo v12

After that I would like to improve Readme file

Do you have some functional documentation?

### PR DESCRIPTION
**Pull request by sseerrggii**
Fix https://github.com/coopiteasy/vertical-cooperative/issues/38

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,8 @@
-# vertical-cooperative
-This repository gather odoo modules for cooperatives
+# Vertical Cooperative
+This project aim to deal with modules related to cooperatives management. You'll find modules that:
+
+ - Allow people sign up as a member of cooperative from website
```
Comment (enricostano) [inline README.md]:
```suggestion
 - Allow people to sign up as a member of the cooperative from website
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,8 @@
...
+ - Manage members' shares, loans and dividends
```
Comment (enricostano) [inline README.md]:
```suggestion
 - Manage member's shares, loans and dividends
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,8 @@
...
+ - Manage cooperative membership
+
+Please don't hesitate to suggest one of your module to this project.
```
Comment (enricostano) [inline README.md]:
```suggestion
Please don't hesitate to suggest one of your modules to this project.
```

**Comment by robinkeunen** [comment]
Thanks @sseerrggii! to comply to OCA, you should add module description to `<module>/readme/DESCRIPTION.rst` like here https://github.com/coopiteasy/vertical-cooperative/pull/84/commits/c93fadbef83b06b71c4b072646d4451e6d9310dd#diff-8386ca81aa0d1c62aea53c58182f278f

**Comment by robinkeunen** [comment]
@sseerrggii ready to integrate your description!

**Comment by sseerrggii** [comment]
Hi @robinkeunen 

Is there any task to be done by me in this PR?

**Comment by robinkeunen** [comment]
Yes, you need to 

1. rebase on the last version of the code 
2. move your contribution to `easy_my_coop/readme/DESCRIPTION.rst`
3. run `oca-gen-addon-readme --repo-name vertical-cooperative --branch 12.0 --org-name coopiteasy --addon-dir easy_my_coop`

If you cannot do **1**, open a new PR. If you cannot do **3**, I'll do it on your new PR. :-)

**Comment by sseerrggii** [comment]
Hey @robinkeunen ! @cesarlr from Coopdevs is helping me! Thanks @cesarlr :heart:

**Comment by robinkeunen** [comment]
closed in favor of https://github.com/coopiteasy/vertical-cooperative/pull/102

**Pull request closed without merging**